glide.lock

```diff
@@ -169,4 +184,63 @@ imports:
   - pkg/util/yaml
   - pkg/watch
   - pkg/watch/versioned
+- name: k8s.io/kubernetes
```

Comment:
I'm still not sure if it is worth it to add it. Especially in cost of adding half of the kubernetes as dependency. I woould like to avaid that if possible.